HW3/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Театр</title>
+	<link rel="stylesheet" href="style.css">
+	<link rel="stylesheet" href="reset.css">
+</head>
+<body>
+	<header class="header" id="header">
+		<div>
+			<h1 class="logo">		<!-- Логотип -->
+				<img src="img/logo.png" alt="logo">
+			</h1>
+		</div>
+			
+		<nav class="main-nav">		<!-- Навигация -->
+			<ul>
+				<li class="active"><a href="#">Головна</a></li>
+				<li><a href="#">Афіша</a></li>
+				<li><a href="#">Репертуар</a></li>
+				<li><a href="#">Новини</a></li>
+				<li><a href="#">Про театр</a></li>
+				<li><a href="#">Персоналії</a></li>
+				<li><a href="#">Контакти</a></li>
+				<li><a href="#">Фестивалі</a></li>
+			</ul>
+		</nav>
+
+		<div class="language">
+			<a href="#">Укр</a>
+		</div>
+	</header>
+
+
+	<section class="slide">  <!-- Карусель -->
+		<img src="img/slide.jpg" alt="Slide" class="slide-photo">
+		<!-- <img src="img/ticket.png" alt="Ticket" class="ticket"> -->
+	</section>
+
+	<div class="main">
+		<nav class="side-menu">  <!-- Боковое меню -->
+			<ul>
+				<li class="icon"><a href="#"><img src="img/photo.png">  Фотоархів</a></li>
+				<li class="icon"><a href="#"><img src="img/video.png">  Відеоархів</a></li>
+				<li class="icon"><a href="#"><img src="img/trustee-board.png" alt="">  Піклувальна рада</a></li>
+				<li class="icon"><a href="#"><img src="img/partners.png">  Парнери</a></li>
+				<li class="icon"><a href="#"><img src="img/forum.png" alt="">  Форум</a></li>
+			</ul>
+		</nav>
+
+		<main>
+			<h2 class="h2">Новини</h2>
+			<div class="article">
+				<article>
+				<img src="img/article.jpg" alt="Article">
+				<h3>Назва новини</h3>
+				<p>
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+				</p>
+				<div class="line"></div>
```

Comment:
```suggestion
				<div class="line"></div>
```
зайвий елемент, краще на бордер на якийсь з вже існуючих))